components/region_cache_memory_engine/src/engine.rs

```diff
@@ -121,6 +133,18 @@ impl RegionMemoryMeta {
     pub fn set_safe_ts(&mut self, safe_ts: u64) {
         self.safe_ts = safe_ts;
```

Comment:
Shall we add an assert that safe_ts must be greater than min_snapshot()?

Reply:
Done